src/components/FilterTags/FilterTags.js

```diff
@@ -1,12 +1,60 @@
 import React, { useState, useRef, useEffect } from "react";
 import classNames from "classnames";
+import { useSelector } from "react-redux";
+import { getModelData } from "app/selectors";
+import {
+  extractCloudName,
+  extractOwnerName,
+  extractCredentialName,
+  pluralize
+} from "app/utils";
 
 import "./_filter-tags.scss";
 
 const FilterTags = () => {
   const [filterPanelVisibility, setfilterPanelVisibility] = useState(false);
   const node = useRef();
 
+  const filters = {};
+
+  const modelData = useSelector(getModelData);
+
+  /**
+  Check if filter exists and adds to array if not
+  @param {string} string The type of filter
+  @param {string} value The name of the filter
+*/
+  const addFilter = function(type, value) {
+    filters[type] = filters[type] || [];
+    if (!filters[type].includes(value)) {
+      filters[type].push(value);
+    }
+  };
+
+  // Loop the model data and pull out the available filters
+  Object.values(modelData).forEach(model => {
```

Comment:
What do you think about moving this filter list generation into a selector? 🤷‍♂

Reply:
Can you explain the benefit?

Reply:
I suppose it would be easier to cache the output. But I'm not convinced, hence the shrug :):